src/alert/alert.style.tsx

```diff
@@ -1,85 +1,145 @@
 // See more https://styled-components.com/docs/api#transient-props
 // =============================================================================
 interface StyleProps {
     $type: AlertType;
+    $size: AlertSizeType;
 }
 
 // =============================================================================
 // STYLING
 // =============================================================================
 
 export const Wrapper = styled.div<StyleProps>`
-    padding: 0.5rem 1rem;
+    padding: 0.5rem 1rem 0.5rem 0.875rem;
     display: flex;
-    flex-direction: column;
     ${TextStyleHelper.getTextStyle("BodySmall", "regular")}
 
     ${(props) => {
         let backgroundColor: string;
         let borderColor: string;
+        let iconColor: string;
+        const iconSize = props.$size === "small" ? "1.25rem" : "1.5rem";
         switch (props.$type) {
             case "error":
                 backgroundColor = Color.Validation.Red.Background(props);
                 borderColor = Color.Validation.Red.Border(props);
+                iconColor = Color.Validation.Red.Icon(props);
                 break;
             case "success":
                 backgroundColor = Color.Validation.Green.Background(props);
                 borderColor = Color.Validation.Green.Border(props);
+                iconColor = Color.Validation.Green.Icon(props);
                 break;
             case "warning":
                 backgroundColor = Color.Validation.Orange.Background(props);
                 borderColor = Color.Validation.Orange.Border(props);
+                iconColor = Color.Validation.Orange.Icon(props);
+                break;
+            case "info":
+                backgroundColor = Color.Validation.Blue.Background(props);
+                borderColor = Color.Validation.Blue.Border(props);
+                iconColor = Color.Validation.Blue.Icon(props);
+                break;
+            case "description":
+                backgroundColor = Color.Neutral[7](props);
+                borderColor = Color.Neutral[4](props);
+                iconColor = Color.Neutral[4](props);
                 break;
             default:
-                backgroundColor = Color.Validation.Green.Background(props);
-                borderColor = Color.Validation.Green.Border(props);
+                backgroundColor = Color.Validation.Orange.Background(props);
+                borderColor = Color.Validation.Orange.Border(props);
+                iconColor = Color.Validation.Orange.Icon(props);
                 break;
         }
 
-        return `
-			background: ${backgroundColor};
-			border-left: 2pt solid ${borderColor};
-		`;
+        return css`
+            background: ${backgroundColor};
+            border-left: 2pt solid ${borderColor};
+
+            svg {
+                color: ${iconColor};
+                width: ${iconSize};
+                height: ${iconSize};
+                margin-right: 0.5rem;
```

Comment:
can I trouble you to add a `flex-shrink: 0;` for the svg so that on smaller widths, it does not get squeezed out by the text